scripts/build.sh

```diff
@@ -75,7 +75,7 @@ function Build {
   if [ -z "$remaining" ]; then
     cmake --build "$BuildDir" --config "$configuration"
   else
-    cmake --build "$BuildDir" --config "$configuration" "${remaining[@]}"
+    cmake --build "$BuildDir" --config "$configuration" $remaining
```

Comment:
The removed quotes here are usually important - it'll stop CMake from treating `/path to/file with spaces.txt` as multiple arguments.